contracts/MintShop1155.sol

```diff
@@ -534,6 +531,16 @@ contract MintShop1155 is Sweepable, ReentrancyGuard {
       _newPaymentReceiver);
   }
 
+
+   /**
+    Allow the shop owner to add new items to the shop,
+
+    @param _items The array of Super1155 addresses.
+  */
+  function addItems(Super1155[] memory _items) external onlyOwner {
```

Comment:
Don't use `onlyOwner` for this; it should be a `PermitControl`-based function that can be delegated to others by the owner.